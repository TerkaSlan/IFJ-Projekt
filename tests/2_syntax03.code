/*This test should throw error because there is not a variable on the left side of assignment */

class Main
{
	static void run()
	{
		int a = 5;
		int b = 3;
		10 = a + b;
	} // end of static void run()
	  	
} // end of class Main