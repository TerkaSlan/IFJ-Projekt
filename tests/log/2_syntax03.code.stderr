ERROR: Syntactic - Line: 9 - Unexpected token.
